frontend/src/pages/RunDetails.tsx

```diff
@@ -868,9 +873,9 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
       const actions = buttons.getToolbarActionMap();
       actions[ButtonKeys.TERMINATE_RUN].disabled =
         (runMetadata.status as NodePhase) === NodePhase.TERMINATING || runFinished;
-      actions[ButtonKeys.RETRY].disabled =
-        (runMetadata.status as NodePhase) !== NodePhase.FAILED &&
-        (runMetadata.status as NodePhase) !== NodePhase.ERROR;
+      // actions[ButtonKeys.RETRY].disabled =
```

Comment:
Please add the same `// BUG(talebz): AIP-6692 WFSDK: Disable Retry button` comment here for consistency.

Reply:
done!